src/libsaml.ts

```diff
@@ -472,7 +472,8 @@ const libSaml = () => {
             metadataCert = [metadataCert];
           }
           metadataCert = metadataCert.map(utility.normalizeCerString);
-          const x509Certificate = select("//*[local-name(.)='X509Certificate']", s)[0].firstChild.data;
+          let x509Certificate = select("//*[local-name(.)='X509Certificate']", s)[0].firstChild.data;
+          x509Certificate = utility_1.default.normalizeCerString(x509Certificate);
```

Comment:
`utility_1.default` is TypeScript compiler output.
In the typescript you'll want to use `x509Certificate = utility.normalizeCerString(x509Certificate)`